custom/rch/views.py

```diff
@@ -0,0 +1,93 @@
+import json
+
+from django.utils.translation import ugettext_noop
+from django.views.generic import TemplateView
+from django.core import serializers
+from django.utils.decorators import method_decorator
+
+from corehq.apps.domain.decorators import require_superuser
+from custom.rch.forms import BeneficiariesFilterForm
+from custom.rch.models import RCHMother, RCHChild
+
+
+class BeneficariesList(TemplateView):
```

Comment:
There would be quite some customisation to this. So would be better to keep it outside admin. Plus might be used by some non-admins as well, later.